### PR TITLE
ops(install): fix rengine install/uninstall

### LINKED ISSUE #37: feat: Add optional removal of images to uninstall.sh script
### Is there an existing feature or issue for this?

- [X] I have searched the existing issues

### Expected feature

Submitted by @AnonymousWP 

Images can take up a lot of space on a machine, especially reNgine's.
Add an optional removal of images in the `uninstall.sh` script, asking the user if it wants to retain images or not.

@AnonymousWP It needs you to recreate your PR here and we could add it to release 2.1.0
 https://github.com/yogeshojha/rengine/pull/888

### Alternative solutions

_No response_

### Anything else?

_No response_

### PR DESCRIPTION
**Pull request by yarysp**
This PR fix broken install/uninstall script

## Fix:
- Container now doing health check on the depending container before starting
I make the install smoother, because while testing I've seen a lot of migration problem due to the db container not starting so fast or concurrently while migrations not complete in other container.

### Now:
1. The db and the redis container are health checked,
2. Then, the celery starts, doing all is long stuff (take ~260s for me to completely run celery-entrypoint.sh)
3. Then, celery-beat and web
4. And, the proxy to finish

This way all container starts well and prevent a lot of install issues.

Eg. It prevents migrations error like this
image

## Fixed:
- uninstall.sh script (fix #37) 
According to this PR https://github.com/yogeshojha/rengine/pull/888
image

- Error ROLE "root" does not exist
role-root-postgres
- Endpoint Techs null error
endpoint-techs-error

All my tests are passed.

### CODE REVIEW

**Comment by yarysp** [comment]
Tested and working, ready to review/merge

**Review by AnonymousWP — DISMISSED** [review]

File: docker-compose.yml [hunk 1]
```diff
@@ -7,16 +7,27 @@ services:
     environment:
       - POSTGRES_DB=${POSTGRES_DB}
       - POSTGRES_USER=${POSTGRES_USER}
+      - PGUSER=${POSTGRES_USER}
```
Comment (AnonymousWP) [inline docker-compose.yml:10]: Wouldn't `PGUSER=${POSTGRES_USER}` make `POSTGRES_USER=${POSTGRES_USER}` redundant you think?

Comment (yarysp) [reply docker-compose.yml:10]: No, the bug fix is like that.
I can find you the source, but the 2 are mandatory if I remeber well

Comment (yarysp) [reply docker-compose.yml:10]: Here
image
https://stackoverflow.com/a/76757417

**Review by AnonymousWP — CHANGES_REQUESTED** [review]

File: docker-compose.yml (hunk 1 above)
Comment (AnonymousWP) [reply docker-compose.yml:10]: Ah okay, thanks.

**Review by AnonymousWP — APPROVED** [review]

**Pull request merged**